Makefile

```diff
@@ -137,33 +140,65 @@ publish-docs:
 	git push -f origin gh-pages
 	git checkout $(CURRENT_BRANCH)
 
-
-# Manage Holochain Conductor config
-HC_LOCAL_STORAGE	= $(shell pwd)/holochain-conductor/storage
-
-.PHONY:		start-hcc-%
-conductor.log:
-	touch $@
-
 dist/holo_hosting_chaperone.js:
 	ln -s node_modules/@holo-host/chaperone/dist dist
 
-check-conductor:	check-holochain
-check-holochain:
-	ps -efH | grep holochain | grep -E "conductor-[0-9]+.toml"
-
-keystore-%.key:
-	@echo "Creating Holochain key for Agent $*: keystore-$*.key";
-	echo $$( hc keygen --nullpass --quiet --path ./keystore-$*.key)			\
-		| while read key _; do							\
-			echo $$key > AGENTID;						\
-		done
-	@echo "Agent ID: $$(cat AGENTID)";
-
 # TMP targets
 use-local-chaperone:
 	yarn uninstall --save @holo-host/chaperone; yarn install --save-dev ../chaperone
 use-yarn-chaperone:
 	yarn uninstall --save @holo-host/chaperone; yarn install --save-dev @holo-host/chaperone
 use-yarn-chaperone-%:
 	yarn uninstall --save @holo-host/chaperone; yarn install --save-dev @holo-host/chaperone@$*
+
+#############################
+# █░█ █▀█ █▀▄ ▄▀█ ▀█▀ █▀▀ ▄▄ █▀ █▀▀ █▀█ █ █▀█ ▀█▀ █▀
+# █▄█ █▀▀ █▄▀ █▀█ ░█░ ██▄ ░░ ▄█ █▄▄ █▀▄ █ █▀▀ ░█░ ▄█
+#############################
+# How to update holochain?
+# In envoy you will have to update the holo-nixpkgs
+# make HOLO_REV="HOLO_REV" HC_REV="" DNA_VERSION="" update-hc
+# Example use: make HOLO_REV="f0e38fd9895054115d8755572e29a5d3639f69e6" update-hc
+# Note: After running this we should run the tests and check
+
+update-hc:
+	make HOLO_REV=$(HC_REV) update-holochain
+	make HOLO_REV=$(HOLO_REV) update-holo-sha
+	make DNA_VERSION=$(DNA_VERSION) update-holo-sha
+	git checkout -b update-hc-$(HC_REV)
+	git add nixpkgs.nix
+	git commit -m hc-rev:$(HC_REV)
+	git push origin HEAD
+
+update-dnas:
+	@if [ $(DNA_VERSION) ]; then\
+		sed -i "24s/.*/  curl 'https:\/\/holo-host.github.io\/holo-hosting-app-rsm\/releases\/downloads\/$(shell echo $(DNA_VERSION) | tr .- _)\/core-app.$(shell echo $(DNA_VERSION) | tr .- _).happ' -o dnas\/holo-hosting-app.happ/" Makefile;\
+		sed -i "26s/.*/  curl 'https:\/\/holo-host.github.io\/servicelogger-rsm\/releases\/downloads\/$(shell echo $(DNA_VERSION) | tr .- _)\/servicelogger.$(shell echo $(DNA_VERSION) | tr .- _).happ' -o dnas\/servicelogger.happ/" Makefile;\
+		sed -i "28s/.*/  curl -LJ 'https:\/\/github.com\/Holo-Host\/dummy-dna\/releases\/download\/v$(DNA_VERSION)\/test.happ' -o dnas\/test.happ/" Makefile;\
```

Comment:
Looks like this new DNA code didn't get reviewed. This sed code is using fixed line numbers right? That seems concerning.

Reply:
We can replace that with a match script